tests/accelerators/test_accelerator_connector.py

```diff
@@ -350,7 +350,6 @@ def test_accelerator_choice_ddp_cpu_and_strategy_spawn(tmpdir):
 
 
 def _test_accelerator_choice_ddp_cpu_and_strategy(tmpdir, ddp_strategy_class):
-    model = BoringModel()
     trainer = Trainer(
         default_root_dir=tmpdir,
         strategy=ddp_strategy_class(find_unused_parameters=True),
```

Comment:
I'm not sure if this is related to the intermittent failures at all, but this test gets a `strategy` AND sets `accelerator="ddp_cpu"` which signals the strategy spawn.

IMO we should change it to `accelerator="cpu"`.

(if done, remember to update the function name)

Reply:
there are many other usages of accelerator="ddp_cpu" and we haven't deprecated that yet (afaik). So we should leave it until that day comes.

Reply:
I'm not saying we need to deprecate it. I mean that with the parameterizations, this becomes

(1):
```python
strategy=DDPSpawnPlugin(),
accelerator="ddp_cpu"  # equals to `strategy="ddp_spawn", accelerator="cpu"`
```
which I guess it's okay since there is no conflict


(2):
```python
strategy=DDPlugin(),
accelerator="ddp_cpu"  # equals to `strategy="ddp_spawn", accelerator="cpu"`
```

where there is a conflict, which one takes precedence, "DDPPlugin" or "ddp_spawn"?

Is the test written like this on purpose?

Reply:
Yes it is. There was a time where we didn't support (2) and I had to add it: #6208
That's basically the test for this functionality.

Reply:
would you mind adding an assertion for the expected TTP class then?

Reply:
I'm not sure what you mean. We have this assertion:

```
assert isinstance(trainer.training_type_plugin, ddp_strategy_class)
```

ddp_strategy_class is either DDPPlugin or DDPSpawnPlugin
while accelerator="ddp_cpu"